cypress/fixtures/project-fix.json

```
@@ -0,0 +1,19 @@
+{
+  "projects": {
+    "25-5-clock": "/pages/25-5-clock",
+    "d3-bar-chart": "/pages/d3-bar-chart",
+    "d3-choropleth": "/pages/d3-choropleth",
+    "d3-heat-map": "/pages/d3-heat-map",
+    "d3-scatter-plot": "/pages/d3-scatter-plot",
+    "d3-tree-map": "/pages/d3-tree-map",
+    "drum-machine": "/pages/drum-machine",
+    "javascript-calculator": "/pages/javascript-calculator",
+    "markdown-previewer": "/pages/markdown-previewer",
+    "portfolio": "/pages/portfolio",
+    "product-landing-page": "/pages/product-landing-page",
+    "random-quote-machine": "/pages/random-quote-machine",
+    "survey-form": "/pages/survey-form",
+    "technical-documentation-page": "/pages/technical-documentation-page",
+    "tribute-page": "/pages/tribute-page"
+  }
```

```suggestion
  "25-5-clock": "/pages/25-5-clock",
  "d3-bar-chart": "/pages/d3-bar-chart",
  "d3-choropleth": "/pages/d3-choropleth",
  "d3-heat-map": "/pages/d3-heat-map",
  "d3-scatter-plot": "/pages/d3-scatter-plot",
  "d3-tree-map": "/pages/d3-tree-map",
  "drum-machine": "/pages/drum-machine",
  "javascript-calculator": "/pages/javascript-calculator",
  "markdown-previewer": "/pages/markdown-previewer",
  "portfolio": "/pages/portfolio",
  "product-landing-page": "/pages/product-landing-page",
  "random-quote-machine": "/pages/random-quote-machine",
  "survey-form": "/pages/survey-form",
  "technical-documentation-page": "/pages/technical-documentation-page",
  "tribute-page": "/pages/tribute-page"
```
Since the rest has been simplified.

---

I tried git ignoring this file because it gets automatically updated, every time.